src/main/java/com/shootoff/camera/cameratypes/PS3EyeCamera.java

```diff
@@ -77,18 +111,28 @@ public static void init() {
 		if (initialized)
 			return;
 
+		try {
+
+			eyecamLib = (eyecam) Native.loadLibrary("eyecam", eyecam.class);
+
+		} catch (UnsatisfiedLinkError exception) {
+
+			initialized = false;
+		}
+
 		eyecamLib.ps3eye_init();
```

Comment:
I think the way you had it before was better, with the following changes:

1. Return after setting initalized false in the catch block

2. Before calling eyecamLib for the first time, make sure it is not null.  If it is null, set initialized false and return.  This is the safest way to handle the possibility that eyecamLib is not what it is supposed to be.  Your current code would not catch a null eyecamLib not resulting from a UnsatisfiedLinkError